src/load-stream-with-many-commits-over-network.bench.ts

```diff
@@ -0,0 +1,46 @@
+import { createCeramic } from "./create-ceramic.js";
+import { TileDocument } from "@ceramicnetwork/stream-tile";
+import { StreamID } from "@ceramicnetwork/streamid";
+import { CeramicApi } from "@ceramicnetwork/common";
+import { scenario } from "./benchie/benchmark.js";
+
+const NUMBER_OF_COMMITS = 100;
+
+scenario("Load stream with many commits over network", (perform) => {
+  let streamId: StreamID;
+  let primaryCeramic: CeramicApi;
+  let secondaryCeramic: CeramicApi;
+
+  perform.beforeAll(async () => {
+    primaryCeramic = await createCeramic(process.env.CERAMIC_ENDPOINT);
+    secondaryCeramic = await createCeramic(
+      process.env.SECONDARY_CERAMIC_ENDPOINT
+    );
+  });
+
+  perform.afterAll(async () => {
+    await primaryCeramic.close();
+    await secondaryCeramic.close();
+  });
+
+  perform.beforeEach(async () => {
+    const content0 = {
+      foo: `hello-${Math.random()}`,
+    };
+    const tile = await TileDocument.create(secondaryCeramic, content0, null, {
+      anchor: false,
```

Comment:
Would be nice to also set publish:false here and below to avoid sending a bunch of pubsub UPDATE messages for each new update.  All the pre-test setup writes to ceramic in all the perf tests should probably be happening with `{ anchor: false, publish:false, pin: true}`

Reply:
tadam: https://github.com/ceramicnetwork/benchie/commit/1655d119a85958c819796159c1f5ac1fea5c905f